scripts/sct_maths.py

```diff
@@ -371,12 +371,11 @@ def main(args = None):
 
     # display message
     if data_out is not None:
-        printv('\nDone! To view results, type:', verbose)
-        printv('fslview ' + fname_out + ' &\n', verbose, 'info')
+        # display viewer syntax
+        sct.display_viewer([fname_out])
     else:
         printv('\nDone! File created: ' + fname_out, verbose, 'info')
 
-
```

Comment:
Change unrelated to issue at hand, not sure if there's a policy for that.

Reply:
no "hard" policy. Not sure we should create one. it's the eternal debate between being clean and save 2min...